docs/tutorials/data_analysis.ipynb

```diff
@@ -7,7 +7,7 @@
         "id": "aA39B_EcZQCc"
       },
       "source": [
-        "##### Copyright 2020 Google"
+        "**Copyright 2020 Google**"
```

Comment:
Ah. Yeah we use this format for our search and strip when importing into the site

Reply:
Ok, I changed sphinx to do this substitution during build time with a simple regex